src/main/java/com/aws/greengrass/componentmanager/plugins/docker/EcrAccessor.java

```diff
@@ -57,7 +57,8 @@ public EcrClient getClient(String region) {
             region = Coerce.toString(deviceConfiguration.getAWSRegion());
         }
 
-        return EcrClient.builder().httpClient(ProxyUtils.getSdkHttpClient())
+        return EcrClient.builder()
+                .httpClientBuilder(ProxyUtils.getSdkHttpClientBuilder().useIdleConnectionReaper(false))
```

Comment:
instead of setting `idleconnectionreaper` here, let's just set it when we make the builder, set it by default.

Reply:
updated